app/helpers/api/ordapi/postOrderItem.test.js

```diff
@@ -96,6 +96,49 @@ describe('postOrderItem', () => {
       });
     });
 
+    it('should post correctly formatted data when there is no service recipient data', async () => {
+      const orderId = 'order1';
+
+      postData.mockResolvedValueOnce({ data: { orderId } });
+
+      const accessToken = 'access_token';
+      const catalogueSolutionId = 'some-solution-id';
+      const orderItemType = 'AssociatedService';
+
+      await postOrderItem({
+        orderId,
+        orderItemId: 'neworderitem',
+        orderItemType,
+        accessToken,
+        serviceRecipientId: null,
```

Comment:
this should be undefined as well / we don't actually need to include them in the method call

Reply:
This inclusion was actually intentional. I wanted to make sure the method worked as expected if one of the service recipient fields was `null` rather than `undefined`. I'm not sure if that should ever happen but I would've thought it is possible (even if unintended – i.e. developer error) so wanted to cover it.